land_vision/urban_tree_detection/log/params.yaml

```diff
@@ -0,0 +1,4 @@
+min_distance: 2
```

Comment:
Should we store these logs in the temp folder so they are not committed? Do we need them?

Reply:
they are used in inference.py, passed to as log directory.  The parmaters control the predicted locs. This was the way the original repository was setup.  I will add a comment in the inference.py main function about how to run this module.